.goreleaser.yml

```diff
@@ -47,6 +47,10 @@ brews:
   description: "Tilt powers multi-service developments for teams that deploy to Kubernetes."
   test: |
     system "#{bin}/tilt version"
+  custom_block: |
+    if  Gem::Specification::find_all_by_name('tilt').any?
```

Comment:
I'm not sure I understand this stanza. can you point us to documentation to better understand how it works?

Reply:
poking around a bit, it seems like keg_only is really for cases like openssl, where end-users know they have to use the absolute path to the openssl installation:
https://docs.brew.sh/Formula-Cookbook#check-for-dependencies

This use-case seems closer to `conflicts_with`: https://docs.brew.sh/Formula-Cookbook#specifying-conflicts-with-other-formulae

Reply:
I'm open to both `keg_only` and `conflicts_with`.

The cookbook mentions that "sometimes there’s hard conflict between formulae, and it can’t be avoided or circumvented with keg_only" and "as a general rule, conflicts_with should be a last-resort option. It’s a fairly blunt instrument." It looks like `conflicts_with` is more escalatory than `keg_only` which is why I chose it.

What would be ideal is having a flag we could pass to `brew install`, like `brew install tilt --keg-only` or `brew install tilt --conflicts-with`. What are your thoughts?

Reply:
haha, ya

The problem is that, AFAICT, keg_only doesn't fix the conflict between the two tilt packages. 

I agree that the language in the documentation just warns you away from conficts_with, without making it super-clear why you'd choose one vs the other

Question: how do _you_ invoke tilt when you have the ruby tilt on your path?

One possibility is to have a second homebrew package that installs it under a different name (tlt? tilt-dev?). goreleaser makes it pretty easy to have multiple homebrew packages

Reply:
Oh `keg_only` _definitely_ fixes the problem. It prevents a link to `/usr/local/bin/tilt` which is where some ruby environments have installed the tilt gem. This then allows users to create a link from opt or the Cellar if they wish.

The project I'm working on (if it were just for my environment, I'd figure out another way of installing tilt--the dev package, not the gem) uses `tlt`. I edited the tilt formula to make it keg_only and then created a symlink from `/usr/local/opt/tilt/bin/tilt` to `/usr/local/bin/tlt`.

Using `tlt` to `tilt-dev` sounds great too! (We're already effectively using `tlt` for all our commands.)